dashboard/client/average_response_time/average_response_time.js

```diff
@@ -15,12 +15,13 @@ Template.averageResponseTime.onRendered(function () {
 
     if (elasticsearchData) {
       // Get aggregations from Elasticsearch data
-      const aggregatedData = elasticsearchData.aggregations.avg_response_time.buckets;
+      const aggregatedData = elasticsearchData.aggregations.requests_over_time.buckets;
```

Comment:
@brylie I'm afraid that I didn't understand the task properly.

I thought that it needs to display average response time which **has granulity "Week".** But the main idea is to display average response time from now to 7 days ago. Please don't look at this, I will re-make it in the separated issue